### PR TITLE
Test Application Sahring

### PR DESCRIPTION
**Pull request by Tharanidk**
Test Case : Test Application Sharing

### CODE REVIEW

**Review by chanaka3d — APPROVED** [review]

File: tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            cy.get('input[name="http://wso2.org/claims/givenname"]').type('user2');
+            cy.get('input[name="http://wso2.org/claims/lastname"]').type('test');
+            cy.get('#password').type(password);
+            cy.get('#password2').type(password);
+            cy.get('input[name="http://wso2.org/claims/emailaddress"]').type('user2@test.com');
+            cy.get('input[name="http://wso2.org/claims/organization"]').type('org1');
+            cy.get('#termsCheckbox').check().should('be.checked');
+            cy.get('#registrationSubmit').click();
+            cy.contains('Close').click();
+
```
Comment (hisanhunais) [inline tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js:86]: Shall we remove the extra line here?

File: tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    const password = 'test123';
+    const carbonUsername = 'admin';
+    const carbonPassword = 'admin';
+    const appName = 'App_A1';
+    const appDescription = 'Testing sharing app ';
+    const groupId = 'org1';
+    const apiName = `anonymous${Math.floor(Math.random() * (100000 - 1 + 1) + 1)}`;
+    const apiVersion = '2.0.0';
+    const apiContext = `anonymous${Math.floor(Math.random() * (100000 - 1 + 1) + 1)}`;
+
```
Comment (hisanhunais) [inline tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js:32]: Shall we remove the extra line here?

File: tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+            cy.get('input[name="http://wso2.org/claims/organization"]').type('org1');
+            cy.get('#termsCheckbox').check().should('be.checked');
+            cy.get('#registrationSubmit').click();
+            cy.contains('Close').click();
+
+
+            //Log into developer portal as user 1
+            cy.loginToDevportal(user1, password);
+
+            //Test with Oath2 Token
```
Comment (hisanhunais) [inline tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js:91]: Shall we correct this "Test with **OAuth2** token"?

File: tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+        //Delete Users
+        cy.loginToPublisher(publisher, password);
+        cy.deleteApi(apiName, apiVersion);
+        cy.visit(`${Utils.getAppOrigin()}/carbon/user/user-mgt.jsp`);
+        cy.deleteUser(user1);
+        cy.deleteUser(user2);
+        cy.deleteUser(publisher);
+        
+    })
+})
```
[the remark this answers is not in the document]
Comment (Tharanidk) [reply tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js:145]: Resolved with commit e159e6db870377cf9dc4726b102cf6292273660e

File: tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+describe("Invoke API Product", () => {
+    const user1 = 'user1';
+    const user2 = 'user2';
+    const publisher = 'publisher';
+    const password = 'test123';
+    const carbonUsername = 'admin';
+    const carbonPassword = 'admin';
+    const appName = 'App_A1';
+    const appDescription = 'Testing sharing app ';
```
[the remark this answers is not in the document]
Comment (Tharanidk) [reply tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js:27]: Resolved with commit e159e6db870377cf9dc4726b102cf6292273660e

File: tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js (hunk 1 above)
Comment (Tharanidk) [reply tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js:86]: done

Comment (Tharanidk) [reply tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js:86]: Resolved with commit dc1fe3cdd1aec56b4bc0534300d8abc77ce18e58

File: tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js (hunk 3 above)
Comment (Tharanidk) [reply tests/cypress/integration/devportal/001-applications/05-test-application-sharing.specs.js:91]: Resolved with commit dc1fe3cdd1aec56b4bc0534300d8abc77ce18e58

**Pull request merged**